lib/sublime-block-comment.js

```diff
@@ -11,15 +11,31 @@ export default {
     const buffer = editor.getBuffer();
 
     buffer.transact(() => {
-      for (const selection of editor.getSelectionsOrderedByBufferPosition().reverse()) {
-        const range = selection.getBufferRange();
-        const textInRange = buffer.getTextInRange(range);
-
+      for (let i in editor.getSelectionsOrderedByBufferPosition().reverse()) {
+        const selection = editor.getSelectionsOrderedByBufferPosition().reverse()[i];
+        const langs = editor.scopeDescriptorForBufferPosition(selection.getBufferRange().start).getScopesArray();
```

Comment:
Can keep `const range = selection.getBufferRange();` above this line and use `const langs = getPositionScopes(range.start);`